.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,35 @@
+name: Stacked App Integration Tests
+
+on: [push, pull_request]
+
+jobs:
+  integration_tests:
+    name: Run stacked app integration tests
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.draft == false
+    defaults:
+      run:
+        working-directory: packages/stacked/example
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Flutter
+        uses: subosito/flutter-action@v1
+        with:
+          channel: "stable"
+
+      - name: Setup chrome driver
+        uses: nanasess/setup-chromedriver@v1
+        with:
+          chromedriver-version: "101.0.4951.41"
+
+      - name: Install dependencies
+        run: flutter pub get
+
+      - name: Launch web driver & run tests
+        run: |
+          chromedriver --port=4444 & 
+          flutter drive \
```

Comment:
I'm curios to know why we're using flutter driver here if the integration_tests package is used to run the tests locally?

Reply:
According to the doc the `integration_tests` package  is compatible with the `flutter drive` command and the command above help us run the test on the web with the help of the chrome driver. My aim was to setup a CI that runs on the web and don't want us to depend on the contributor local test alone.

<img width="759" alt="Screenshot 2022-05-29 at 11 30 34" src="https://user-images.githubusercontent.com/46024202/170863537-30e8e7ef-f0a1-43e4-bc47-6d0fc2ff2424.png">

This is also the CI on my fork branch
![image](https://user-images.githubusercontent.com/46024202/170864032-604099f1-8df1-4f16-9e26-308f0a45da2c.png)

For the new commits, incase a contributor just want to use their terminal for testing, it uses the `flutter driver` command.
Here is demo for it 
<a href="https://www.loom.com/share/94e4e5e49dfa48389188502d0f63dfcc">
    <p>Running with chrome driver on CLI - Watch Video</p>
    <img style="max-width:300px;" src="https://cdn.loom.com/sessions/thumbnails/94e4e5e49dfa48389188502d0f63dfcc-with-play.gif">
  </a>